src/views/catalog/utils/quick-create-vm.ts

```diff
@@ -27,13 +31,21 @@ type QuickCreateVMType = (inputs: {
     namespace: string;
     startVM: boolean;
     subscriptionData: RHELAutomaticSubscriptionData;
+    templateBootSource: TemplateBootSource;
   };
   template: V1Template;
 }) => Promise<V1VirtualMachine>;
 
 export const quickCreateVM: QuickCreateVMType = async ({
   models,
-  overrides: { authorizedSSHKey, name, namespace = DEFAULT_NAMESPACE, startVM, subscriptionData },
+  overrides: {
```

Comment:
the override getting bloated, is there another way to resolve it?

Reply:
refactored quickCreate to apply all overrides in the onQuickCreate method, for some reason we were modifying the VM before and after processing the template which looked weird to me, tested multiple use cases should work the same as before.